src/link/abstractlink.h

```diff
@@ -251,5 +275,22 @@ class AbstractLink : public QObject
 private:
     bool _autoConnect;
     QString _name;
+    QTimer _oneSecondTimer;
     LinkType _type;
+
+    // Up and down speed logic
+    struct BitRateSpeed {
+        float speed;
+        float numberOfBytes;
+
+        /**
+         * @brief Reset total number of bytes and set the current speed
+         *
+         */
+        void update()
+        {
+            speed = numberOfBytes;
```

Comment:
`speed = numberofBytes*8;`

maybe refactor `speed` -> `bitsPerSecond`

Reply:
It would be better for us to work with bytes as a consistent unit, there is no advantage to use bits directly since our communication is bytes based and the conversion between both units are pretty simple. 
Also bits does not reflect directly in information, since our main unit for "data" are byte arrays from 0-255 for our sonars, it makes more sense to use bytes as a speed representation, otherwise we'll need to divide by 8 to understand what is going on.

Reply:
Let me know what you think.

Reply:
In my mind I would be comparing the number to the 115200 baud or 1Mbaud that we have configured to think about saturation, and the bandwidth measurements with iperf or companion to see the bandwidth comparision/contribution. I think you are right for your own reasons. It's nbd to convert.